app/src/main/java/com/simplecity/amp_library/model/ArtworkModel.java

```diff
@@ -42,9 +42,7 @@ public boolean equals(Object o) {
 
         ArtworkModel that = (ArtworkModel) o;
 
-        if (type != that.type) return false;
-        return file != null ? file.equals(that.file) : that.file == null;
-
+        return type == that.type && (file != null ? file.equals(that.file) : that.file == null);
```

Comment:
I'd prefer not to make this change. 'Simplifying' the conditional statement generally makes it harder to read for equals() method.